releasenotes-dca/notes/add-option-to-enable-crd-collection-0f1f55f9a34ff1f5.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Switch CRD collector to stable collector
```

```suggestion
    Enable CRD collection
```

nit: change the release note filename for consistency